CONTRIBUTING.md

```diff
@@ -1,54 +1,151 @@
-# Contributing
+# Guidelines for Contributing
 
-This document contains information about setting up a development environment for contributing to the project.
+PyMC-Marketing welcomes contributions from interested individuals or groups. These guidelines are provided to give potential contributors information to make their contribution compliant with the conventions of the PyMC-Marketing project, and maximize the probability of such contributions to be merged as quickly and efficiently as possible.
```

Comment:
Maybe something along the lines:
*You do not need to be an expert to contribute (see below), just be interested and willing to learn and share knowledge.* 
To make the guidelines more welcoming.

Reply:
I've added something along these lines